### PR TITLE
Added missing csharp options in tspconfig.yaml for Qumulo

### PR DESCRIPTION
**Pull request by scorpion-ajay**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

<a href="?expand=1&template=sdk_configuration_template.md">Click here</a> to open a PR for only SDK configuration.

### CODE REVIEW

File: specification/liftrqumulo/Qumulo.Storage.Management/tspconfig.yaml [hunk 1]
```diff
@@ -24,6 +24,13 @@ options:
     flavor: "azure"
     packageDetails:
       name: "@azure/arm-qumulo"
+  "@azure-tools/typespec-csharp":
+    emitter-output-dir: "{project-root}/azure-sdk-for-net/sdk/qumulo/{namespace}/src"
+    package-dir: "Azure.ResourceManager.Qumulo"
+    service-dir: "sdk/qumulo"
```
Comment (chunyu3) [inline specification/liftrqumulo/Qumulo.Storage.Management/tspconfig.yaml:30]: `service-dir` is not emitter option. It is a parameter.. please remove it.

File: specification/liftrqumulo/Qumulo.Storage.Management/tspconfig.yaml [hunk 2]
```diff
@@ -24,6 +24,13 @@ options:
     flavor: "azure"
     packageDetails:
       name: "@azure/arm-qumulo"
+  "@azure-tools/typespec-csharp":
+    emitter-output-dir: "{project-root}/azure-sdk-for-net/sdk/qumulo/{namespace}/src"
```
Comment (chunyu3) [inline specification/liftrqumulo/Qumulo.Storage.Management/tspconfig.yaml:28]: We will not specify the `emitter-output-dir` here. We will calculate from `service-dir` and `package-dir`

**Comment by live1206** [comment]
@msyyc Please check the Python breaking change, should not be caused by this PR

File: specification/liftrqumulo/Qumulo.Storage.Management/tspconfig.yaml (hunk 1 above)
Comment (chunyu3) [reply specification/liftrqumulo/Qumulo.Storage.Management/tspconfig.yaml:29]: sorry. `service-dir` can define under emitter option to override the global one. So we can keep this.

**Review by dipeshbhakat-microsoft — APPROVED** [review]

**Pull request merged**